tasks/add_picklist_values.py

```diff
@@ -0,0 +1,306 @@
+# Optionally updates the picklist values to be alphabetical.
+class AddPicklistValues(BaseSalesforceApiTask, Deploy):
+    task_options = {
+        "sobject": {
+            "description": "SObject of the picklist being modified",
+            "required": True,
+        },
+        "field": {
+            "description": "The picklist being modified",
+            "required": True,
+        },
+        "values": {
+            "description": "A comma-delimited list of the picklist values being added",
+            "required": True,
+        },
+        "recordtypes": {
+            "description": "The record types for which these new picklist values will be available",
+            "required": False,
+        },
+        "sorted": {
+            "description": "If true, sets the entire picklist to be sorted in alphabetical order",
+            "required": False,
+        },
+        "otherlast": {
+            "description": "If true, the Other value (if one exists), will remain at the end of the values",
+            "required": False,
+        }
+    }
+
+    def _run_task(self):
+        sobject = self.options["sobject"]
+        field = self.options["field"]
+        values = self.options["values"].split(',')
+
+        # The Tooling API does not have the namespace or the '__c' suffix in the DeveloperName of the CustomField object,
+        # so we have to remove it.
+        split_field = field.split('__')
+        if len(split_field) is 3:
+            field_developer_name = split_field[1]
+            namespace = split_field[0]
+        else:
+            field_developer_name = split_field[0]
+            namespace = ""
+
+        field_tooling_results = self.tooling.query(
+            "SELECT Id, DeveloperName, Metadata "
+            "FROM CustomField "
+            "WHERE NamespacePrefix = '{namespace}' "
+            "AND DeveloperName = '{developer_name}' ".format(namespace = namespace, developer_name = field_developer_name)
+        )
+
+        if field_tooling_results["size"] is not 1:
+            raise Exception('{} field not found on {}'.format(field, sobject))
+
+        validated_field = field_tooling_results["records"][0]["Metadata"]
+        existing_picklist_values = []
+
+        # build up a list of active record types
+        picklist_record_types = self._get_active_record_types(sobject)
+
+        if validated_field["type"] == "Picklist":
+            existing_picklist_values = validated_field["valueSet"]["valueSetDefinition"]["value"]
+        else:
+            raise Exception('{} field is not a picklist field'.format(field))
+
+        # remove any existing picklist values from the list of picklist values to add
+        for existing_picklist_value in existing_picklist_values:
+            if existing_picklist_value["label"] in values:
+                print('{} is already a picklist value on {}. Skipping...'.format(existing_picklist_value["label"], field))
```

Comment:
For logging statements you should use `self.logger`:
```suggestion
                self.logger.info(f'{existing_picklist_value["label"]} is already a picklist value on {field}. Skipping...')
```
I also converted it to a [f-string](https://docs.python.org/3/reference/lexical_analysis.html#f-strings).

Reply:
Thanks! Will add in my next commit as I've got a few other changes to commit up as well.